usermods/usermod_v2_four_line_display_ALT/readme.md

```diff
@@ -74,6 +62,10 @@ These options are configurable in Config > Usermods
 
 Note: the Four Line Display usermod requires the libraries `U8g2` and `Wire`.
 
+## Compatibility
+
+The original "usermod_v2_auto_save" will not work with the display just yet.
```

Comment:
It worked when I last checked.

Reply:
I didn't modified it. In the readme it was written that autosave usermod is not supported, so I simply reported it.
Should I remove this line?